server/pages/index.jade

```diff
@@ -112,7 +115,7 @@ html(lang=lang, dir=isRTL ? 'rtl' : 'ltr', class=isFluidWidth ? 'is-fluid-width'
 			else
 				if sectionCss
 					link(rel='preload' as='style' href=sectionCss)
-		
+
 		if shouldUseStylePreloadCommon
 			if isRTL
 				link(rel='preload' as='style' href=urls['style-rtl.css'])
```

Comment:
Shouldn't we also update that part of the code as well?

We have other [`.jade templates`](https://github.com/Automattic/wp-calypso/tree/master/server/pages) that could benefit from this change as far I can see.

Reply:
I wasn't aware of other pages, thanks! I'll search over the whole codebase now for stylesheet links